spec/indexers/hyrax/file_set_indexer_spec.rb

```diff
@@ -43,7 +43,7 @@
 
   let(:resource_version) do
     ActiveFedora::VersionsGraph::ResourceVersion.new.tap do |v|
-      v.uri = "http://test.example/rest/example/foo123/files/#{mock_file.id}/fcr:versions/version1"
+      v.uri = "#{file_set.uri}/files/#{mock_file.id}/fcr:versions/version1"
```

Comment:
using an actual id here, and not stubbing one that's deliberately shorter than the real implementation, fixes this test.